evaluation/swe_bench/README.md

```diff
@@ -85,7 +86,7 @@ If you see an error, please make sure your `config.toml` contains all
 ## Run Inference on SWE-Bench Instances
 
 ```bash
-./evaluation/swe_bench/scripts/run_infer.sh [model_config] [git-version] [agent] [eval_limit]
+./evaluation/swe_bench/scripts/run_infer.sh [model_config] [git-version] [agent] [eval_limit] [max_iter] [num_workers] [use_instance_image] [instance_docker_namespace]
```

Comment:
How about we make "[use_instance_image]" an environment variable and default to false, similar to the `USE_HINT_TEXT`: https://github.com/OpenDevin/OpenDevin/blob/4049c69590f8f256478eb64c9c6b2003da87812d/evaluation/swe_bench/scripts/run_infer.sh#L42-L44

We can completely remove the `[instance_docker_namespace]` option, since we already "re-tag" the instances to remove their namespaces when pulling? Or do you think we should keep it, in case the user forgets to run the `pull` script?- if so, we can make it environment var as well.

https://github.com/OpenDevin/OpenDevin/blob/4049c69590f8f256478eb64c9c6b2003da87812d/evaluation/swe_bench/scripts/docker/pull_all_eval_docker.sh#L46

Reply:
Let's make it an env var + remove instance_docker_namespace!